doc/content/source/markers/MinimumElementSizeMarker.md

```diff
@@ -4,9 +4,9 @@
 
 ## Description
 
-A marker that flags elements for refinement based on element length. The element size is provided to this object via
-the "indicator" parameter. If an element has a minimum length less than the minimum element defined by the Indicator
-then the element is marked for refinement, otherwise it is not marked.
+A marker that flags elements for refinement based on element length. The element size is provided to this object either
+via the "indicator" parameter or an "element_size" value. If an element has a minimum length greater than the minimum element defined by either the Indicator or the Element Size, then the element is marked for refinement,
```

Comment:
Please proofread this again.

Reply:
If an element has a minimum dimension greater than the size defined by either the Indicator or the `element_size` input parameter, then the element is marked for refinement,